### PR TITLE
HAWQ-423. Updated gradle and download plugin version

### PR DESCRIPTION
**Pull request by shivzone**

### CODE REVIEW

**Comment by GodenYao** [comment]
+1

File: pxf/build.gradle [hunk 1]
```diff
@@ -434,7 +435,6 @@ task tomcatRpm(type: Rpm) {
         packageName 'apache-tomcat'
         summary = 'Apache Tomcat RPM'
         vendor = project.vendor
-        release = buildNumber()
```
Comment (sansanichfb) [inline pxf/build.gradle]: Why do we need to remove build number from rpm name?

Comment (shivzone) [inline pxf/build.gradle]: The HAWQ pulse build number is not relevant for the apache tomcat rpm.
Besides we are not releasing internal custom builds of apache tomcat

**Comment by sansanichfb** [comment]
+1

**Pull request closed without merging**